src/core/Object3D.js

```diff
@@ -332,7 +332,7 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 			position.setFromMatrixPosition( this.matrixWorld );
 
-			if ( this.isCamera ) {
+			if ( this.isCamera || this.isLight ) {
```

Comment:
This change is scary, but less see what happens...

Reply:
We'll see... Thanks for giving this PR a go!